execution/src/vm.rs

```diff
@@ -44,16 +44,16 @@ fn call(shared_env: &SharedExecutionContext, addr: Address, func_name: String, m
 
         // update context
         (*exec_context_guard).max_gas = max_gas;
-        (*exec_context_guard).coins = 0; // TODO maybe allow sending coins in the call
+        (*exec_context_guard).coins = Amount::from_raw(0); // TODO maybe allow sending coins in the call
         (*exec_context_guard).call_stack.push_back(addr);
     }
 
     // run
     let mut run_failed = false;
-    match Instance::new(&target_module, &todo_imports) // TODO bring imports into the execution context (?)
-        .and_then(|inst| inst.exports.get_function(func_name))
-        .and_then(|f| f.native::<(), ()>()) // TODO figure out the "native" explicit parameters
-        .and_then(|f| f.call())
+    match Instance::new(&target_module, &ImportObject::new()) // TODO bring imports into the execution context (?)
```

Comment:
Changes in this file are just quick changes to make it compile... @damip

Reply:
Whatever, this part of the code will certainly change when we'll do the integration of the engine :+1: